src/nvim/version.c

```diff
@@ -14,13 +14,13 @@
 #include "nvim/ascii.h"
 #include "nvim/buffer.h"
 #include "nvim/charset.h"
+#include "nvim/drawscreen.h"
```

Comment:
Why have you changed a lot of these includes? Surely include `screen.h` is more preferable since it pulls in much less? That's how I designed my first first draft of this.

Reply:
Because I removed `drawscreen.h` from `buffer.h`. This previously included `drawscreen.h` indirectly through `buffer.h`.